numpy/lib/function_base.py

```diff
@@ -2965,7 +2957,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
 
     >>> x = [-2.1, -1,  4.3]
     >>> y = [3,  1.1,  0.12]
-    >>> X = np.vstack((x,y))
+    >>> X = np.concatenate([np.atleast_2d(m) for m in (x,y)], axis=0)
```

Comment:
I was trying to write general code. stack works in this case for 1d arrays and then concatenate is needed for 2d arrays. I changed it to stack though in 50514ed